lib/DDG/Spice/PackageTracking.pm

```diff
@@ -74,6 +74,43 @@ triggers query_nowhitespace => qr/^
                                 )
                                 $/xi;
 
+## CanadaPost
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    [\d]{9,} |
+                                    [A-Z]{2}\d{9}CA)
+                                )
+                                $/xi;
+
+## DHL
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    \d{9,} |
```

Comment:
@Owlree okay, let's go with 10 digits for now. If we get complaints about the IA not triggering, we can update accordingly.